test/kernel_acl.js

```diff
@@ -11,14 +11,15 @@ contract('Kernel ACL', accounts => {
     const granted = accounts[1]
     const child = accounts[2]
 
-    const action = getSig('upgradeKernel(address)')
+    let role = ''
```

Comment:
Is empty string required or could it be null?

Reply:
Could be null actually